langchain/retrievers/weaviate_hybrid_search.py

```diff
@@ -40,6 +40,18 @@ def __init__(
         if attributes is not None:
             self._query_attrs.extend(attributes)
 
+        self._create_schema()
+
+    def _create_schema(self) -> None:
```

Comment:
It might make sense to rename to explain that this runs conditionally on the schema being missing.

```suggestion
    def _create_schema_if_missing(self) -> None:
```

Should any of the schema parameters be exposed? e.g., the vectorizer?

Reply:
> Should any of the schema parameters be exposed? e.g., the vectorizer?

Given the way the class is designed, I think the only parameter that should be exposed is the vectorizer. However, we could also use langhcain to the vectorization instead of assuming that the user has set up weaviate with the vectorizer module enabled. Which approach would you like to go with?

Reply:
I generally think that good defaults with flexibility is better for usage. :)

Reply:
The usage we've seen in other frameworks are that they will use other the framework's embeddings feature to generate the embeddings. The vector db is meant to just store and retrieve the embeddings.

Let's keep this like this for now and reconsider again based on user feedback. E.g. if a user wants to use GPT4All for the embeddings then we definitely need to rewrite this class.